Nodejs/Product/InteractiveWindow/InteractiveWindow.csproj

```diff
@@ -164,6 +141,151 @@
     <None Include="Theme\Microsoft.NodejsTools.InteractiveWindow.theme.v15.0.pkgdef" />
     <None Include="Theme\make_theme_pkgdef.py" />
   </ItemGroup>
+  <!-- only include the loc files for VS 15 -->
+  <ItemGroup Condition="$(VSTarget) == '14.0'">
+    <VSCTCompile Include="InteractiveWindow.vsct">
+      <ResourceName>Menus.ctmenu</ResourceName>
+      <SubType>Designer</SubType>
+    </VSCTCompile>
+    <Compile Include="Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+    <EmbeddedResource Include="Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+      <SubType>Designer</SubType>
+    </EmbeddedResource>
+    <ConditionalEmbeddedResource Include="VSPackage.resx">
+      <MergeWithCTO>true</MergeWithCTO>
+      <ManifestResourceName>Microsoft.VSPackage</ManifestResourceName>
+      <SubType>Designer</SubType>
+    </ConditionalEmbeddedResource>
+  </ItemGroup>
+  <ItemGroup Condition="$(VSTarget) == '15.0'">
+    <VSCTCompile Include="InteractiveWindow.cs.vsct">
+      <ResourceName>Menus.ctmenu</ResourceName>
```

Comment:
Not super familiar with this stuff... but it's OK to add all language versions of the .vsct with the same "ResourceName"? Does the locale from the included item persist by some other means so the right one is selected at runtime?